gdp/column/column.py

```diff
@@ -25,21 +25,25 @@ def build_column(min_trays, max_trays, xD, xB, x_input, nlp_solver, provide_init
     t_start = time.process_time()
     """Builds the column model."""
     m = ConcreteModel('benzene-toluene column')
-    m.comps = Set(initialize=['benzene', 'toluene'])
-    min_T, max_T = 300, 400
-    m.T_ref = 298.15
-    max_flow = 500
+    m.comps = Set(initialize=['benzene', 'toluene']) ## Initialize component set
+    min_T, max_T = 300, 400  ## Define temperature range
+    m.T_ref = 298.15 ## Define reference temperature
+    max_flow = 500 ## Define maximum flow rate
+    ## Define number of trays, condenser and reboiler trays
     m.max_trays = max_trays
     m.condens_tray = max_trays
     m.feed_tray = math.ceil((max_trays / 2))
     m.reboil_tray = 1
+    ## Define distillate and bottoms purity
```

Comment:
Why are you using ##? In other files, you use a single one. Please homogenize it. I prefer a single one following black formatting https://dev.to/adamlombard/how-to-use-the-black-python-code-formatter-in-vscode-3lo0

Reply:
I removed the one of it.